deploy/k8s/charts/trustification/templates/init/dataset/020-Job.yaml

```diff
@@ -39,25 +40,65 @@ spec:
           args:
             - "-ce"
             - |
-              ARGS=""
+              ARGS="--report-enable {{$reportEnable}} "
```

Comment:
Assuming we would enable clap args, as suggested before, we would not need to craft argument strings, but could use env-vars to set this directly.

Same for the other locations.